pkg/services/live/live.go

```diff
@@ -0,0 +1,187 @@
+package live
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/centrifugal/centrifuge"
+	"github.com/grafana/grafana/pkg/infra/log"
+	"github.com/grafana/grafana/pkg/models"
+)
+
+var (
+	logger   = log.New("live")
+	loggerCF = log.New("live.centrifuge")
+)
+
+// GrafanaLive pretends to be the server
+type GrafanaLive struct {
+	node    *centrifuge.Node
+	Handler interface{} // handler func
+}
+
+// InitalizeBroker the broker and start listening for requests
+func InitalizeBroker() (*GrafanaLive, error) {
+	// We use default config here as starting point. Default config contains
+	// reasonable values for available options.
+	cfg := centrifuge.DefaultConfig
+
+	// cfg.LogLevel = centrifuge.LogLevelDebug
```

Comment:
Should we keep this commented out code?

Reply:
Keeping it as debug is too much when channels are publishing fast, but it is useful to have at debug to understand which events get hit in which order